stable/insights-agent/download-kubectl.sh

```diff
@@ -1,29 +1,41 @@
 #! /bin/sh
 
-set -e
+set -eo pipefail
```

Comment:
I tried adding the `-o pipeline` here but it doesn't seem to like it. It doesn't error out but it says "Invalid option" or something

Reply:
@mhoss019 it's `pipefail` not `pipeline` 😄 

Seems to be OK now - anything I should check besides that the tests are passing?

Reply:
Yeah it doesn't like this. If you look at the logs it has "illegal option"

Reply:
OOOOOO my bad! Looks good. Thanks!

Reply:
That's from the previous run 😆 

```
sh: set: line 1: illegal option -o pipeline
```
now it's `pipefail`